samples/build-cache/settings.gradle.kts

```diff
@@ -0,0 +1,21 @@
+buildCache {
+    local {
+        isEnabled = true
+    }
+    remote(DirectoryBuildCache::class.java) {
+        setDirectory("path/to/some/local-cache")
+        isEnabled = true
+        isPush = false
+        targetSizeInMB = 1024
+    }
+    remote(HttpBuildCache::class.java) {
```

Comment:
This API could use some reified type love.

Not a part of this PR though.

Reply:
Indeed.

Reply:
That will be part of #221.